packages/core/src/types/IVolume.ts

```diff
@@ -20,7 +20,7 @@ interface IVolume {
   /** volume direction */
   direction: Mat3;
   /** volume scalarData */
-  scalarData: Float32Array | Uint8Array;
+  scalarData?: Float32Array | Uint8Array;
```

Comment:
I don't like how we are making this optional, since you are gathering the scalarData inside `timePointsData`. To me we should have the same interface, but change `imageIds` to `[] | [][]` and same for `scalarData`

Reply:
I tried changing imageIds from `string[]` to `string[][]` but I started facing issues outside of the volume context because some functions access `imageIds` directly (eg: [cache](https://github.com/cornerstonejs/cornerstone3D-beta/blob/main/packages/core/src/cache/cache.ts#L463-L475))